packages/netlify/package.json

```diff
@@ -42,13 +42,13 @@
     "esbuild": "^0.19.5"
   },
   "peerDependencies": {
-    "astro": "^3.0.0"
+    "astro": "^3.0.0 | ^4.0.0"
```

Comment:
It seems this should be two pipes (`"astro": "^3.0.0 || ^4.0.0"`) as per the [npm package.json docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies). Preventing upgrade to Astro 4: see #99.